### PR TITLE
[WEBSITE-434] Show basic step description when detailed check fails

### PR DESCRIPTION
**Pull request by zbynek**
See https://issues.jenkins-ci.org/browse/WEBSITE-434

Make sure step reference creates at least the basic description for the special `parallel` step.

### CODE REVIEW

File: src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java [hunk 1]
```diff
@@ -146,16 +151,27 @@ public static String generateStepHelp(QuasiDescriptor d){
...
-        } catch(Exception ex){
-            ex.printStackTrace();
-            mkDesc.append("<code>").append(ex).append("</code>\n\n");
-        } catch(Error err){
-            err.printStackTrace();
-            mkDesc.append("<code>").append(err).append("</code>\n\n");
+            try {
+                mkDesc.append(generateHelp(new DescribableModel(d.real.clazz), true));
+            } catch (Exception ex) {
+                mkDesc.append(getHelp("help.html", d.real.clazz));
```
Comment (kwhetstone) [inline src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java]: Not entirely sure where this is; is this in the directory as an alternative?

File: src/main/java/org/jenkinsci/pipeline_steps_doc_generator/PipelineStepExtractor.java [hunk 2]
```diff
@@ -158,8 +158,9 @@ public static void main(String[] args){
                                     Class<?> delegateOptionType = delegateOptionSchema.getType();
                                     Descriptor<?> delegateDescriptor = Jenkins.getInstance().getDescriptor(delegateOptionType.asSubclass(Describable.class));
                                     // TODO for some reason, stepsToPlugin contains entries for descriptors but not their describables:
```
Comment (kwhetstone) [inline src/main/java/org/jenkinsci/pipeline_steps_doc_generator/PipelineStepExtractor.java]: Update comment if stripping out stepsToPlugin, and explain what's going on here

File: src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java (hunk 1 above)
Comment (zbynek) [reply src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java]: It's already used in the "default" case by describable model, I just had to copy DecribableModel explicitly as we can't always create an instance of it.

File: src/main/java/org/jenkinsci/pipeline_steps_doc_generator/PipelineStepExtractor.java (hunk 2 above)
Comment (zbynek) [reply src/main/java/org/jenkinsci/pipeline_steps_doc_generator/PipelineStepExtractor.java]: The comment is now removed and there are some comments for the new code.

**Comment by kwhetstone** [comment]
LGTM, thanks for adding the comments.

**Pull request merged**